agenda-services/src/main/java/org/exoplatform/agenda/job/ComputingReminderJob.java

```diff
@@ -0,0 +1,68 @@
+package org.exoplatform.agenda.job;
+
+import org.exoplatform.agenda.constant.EventAttendeeResponse;
+import org.exoplatform.agenda.exception.AgendaException;
+import org.exoplatform.agenda.model.Event;
+import org.exoplatform.agenda.model.EventFilter;
+import org.exoplatform.agenda.model.EventReminder;
+import org.exoplatform.agenda.service.AgendaEventReminderService;
+import org.exoplatform.agenda.service.AgendaEventService;
+import org.exoplatform.commons.utils.CommonsUtils;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+import org.quartz.*;
+import org.quartz.impl.StdSchedulerFactory;
+
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ComputingReminderJob implements Job {
+  private static final Log   LOG = ExoLogger.getLogger(ComputingReminderJob.class);
+
+  private AgendaEventService eventService;
+
+  private AgendaEventReminderService eventReminderService;
+
+  @Override
+  public void execute(JobExecutionContext context) throws JobExecutionException {
+    eventService = CommonsUtils.getService(AgendaEventService.class);
+    eventReminderService = CommonsUtils.getService(AgendaEventReminderService.class);
+    try {
+      LOG.info("/********************** job invoked ******************************/");
```

Comment:
not an informative log. This is useless decoration for an ops. please remove or replace by something more useful.